spec/controllers/cards_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+describe Dashboard::CardsController do
+  describe 'parsing_html' do
+    before do
+      @user = create(:user)
+      @block = create(:block, user: @user)
+      @controller.send(:auto_login, @user)
+    end
+
+    it 'add delayed job' do
+      post :parsing_html, { format: 'js',
+                            parsing_html:
+                              { block_id: @block.id,
+                                url: 'http://www.learnathome.ru/blog/100-beautiful-words',
+                                origin_selector: 'table tbody tr td[2] p',
+                                translated_selector: 'table tbody tr td[1] p' }}
+      expect(Delayed::Job.count).to eq(1)
+      expect(flash[:notice]).to eq('Задача на парсинг сайта успешно поставлена в очередь.')
+      expect(flash[:alert]).to eq(nil)
+    end
+
+    it 'incorrect url' do
+      post :parsing_html, { format: 'js',
+                            parsing_html:
+                              { block_id: @block.id,
+                                url: 'http://www.learnathome.ru/blog/100-beautiful',
+                                origin_selector: 'table tbody tr td[2] p',
+                                translated_selector: 'table tbody tr td[1] p' }}
+      expect(Delayed::Job.count).to eq(0)
+      expect(flash[:alert]).to eq('Проверьте правильность введенных данных.')
+      expect(flash[:notice]).to eq(nil)
+    end
+
+    it 'incorrect css original' do
+      post :parsing_html, { format: 'js',
+                            parsing_html:
+                              { block_id: @block.id,
+                                url: 'http://www.learnathome.ru/blog/100-beautiful-words',
+                                origin_selector: 'table tbody',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.